aries-site/src/examples/templates/content-layouts/ContentDrivenLayout.js

```diff
@@ -0,0 +1,111 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  Form,
+  FormField,
+  Notification,
+  Paragraph,
+  Text,
+  TextInput,
+} from 'grommet';
+
+import { ModalBody, ModalDialog, ModalFooter } from '../ModalDialog';
+
+const path = '/products/data/47287498729/collections/KCHDvfcByKvvjymNheg';
+const parts = path.split('/').map(part => `/${part}`);
+const matchPath = [
+  ...parts.map(part => ({
+    regexp: new RegExp(part),
+    message: `Location paths do not match. Expecting ${part}.`,
```

Comment:
Should the error be reading the whole path?
<img width="530" alt="Screen Shot 2022-04-22 at 10 24 47 AM" src="https://user-images.githubusercontent.com/12522275/164764201-f98c4753-b5dd-4001-9714-81c0e43015c4.png">
```suggestion
    message: `Location paths do not match. Expecting ${path}.`,
```

Reply:
Or is it common behavior to break these paths up? Was surprised when the solution to the error didn't fully resolve the error.

Reply:
With such a long string, I was attempting to try to help the user identify the problem. I think this could be resolved by changing the message to read:

```
... Expecting ${path}. Missing ${part}.
```

Would that help?

Reply:
Yes that might help, was confusing I think in my case because i was missing multiple parts (just typed in one character) which is less likely. I think rephrasing just to `Missing ${part}` might even be sufficient. (but we should probably loop this example in with the broader pattern of "double confirmations")

Reply:
Yeah, there's an issue for that. Rena was going to do some research for GLCP and share her findings and explorations. I thought I'd play around with it as well since the topic had come up recently.

Reply:
done